<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="kemas" failures="0" tests="7" name="schemaTesting.Schema_Table_Testing" time="0.238" errors="1" timestamp="2022-12-13T13:00:21 IST" skipped="0">
  <testcase classname="schemaTesting.Schema_Table_Testing" name="test_table_NumOfCol" time="0.114"/>
  <system-out/>
  <testcase classname="schemaTesting.Schema_Table_Testing" name="test_table_ColNames" time="0.031">
    <error message="After end of result set" type="java.sql.SQLException">
      <![CDATA[java.sql.SQLException: After end of result set
at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:129)
at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:97)
at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:89)
at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:63)
at com.mysql.cj.jdbc.result.ResultSetImpl.checkRowPos(ResultSetImpl.java:532)
at com.mysql.cj.jdbc.result.ResultSetImpl.getString(ResultSetImpl.java:878)
at com.mysql.cj.jdbc.result.ResultSetImpl.getString(ResultSetImpl.java:896)
at schemaTesting.Schema_Table_Testing.test_table_ColNames(Schema_Table_Testing.java:55)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- test_table_ColNames -->
  <system-out/>
  <testcase classname="schemaTesting.Schema_Table_Testing" name="test_table_ColNames_Datatype" time="0.024"/>
  <system-out/>
  <testcase classname="schemaTesting.Schema_Table_Testing" name="test_table_SizeOfColNames" time="0.018"/>
  <system-out/>
  <testcase classname="schemaTesting.Schema_Table_Testing" name="test_table_nullFields" time="0.018"/>
  <system-out/>
  <testcase classname="schemaTesting.Schema_Table_Testing" name="test_table_columnkeys" time="0.010"/>
  <system-out/>
  <testcase classname="schemaTesting.Schema_Table_Testing" name="test_tablename_tableschema" time="0.023"/>
  <system-out/>
</testsuite> <!-- schemaTesting.Schema_Table_Testing -->
